classes/Tools.php

```diff
@@ -3466,6 +3466,72 @@ public static function arrayReplaceRecursive($base, $replacements)
         }
         return $base;
     }
+
+    /**
+     * @deprecated Deprecated since 1.7.0
+     * Return path to a Product or a CMS category
+     *
+     * @param string $url_base Start URL
+     * @param int $id_category Start category
+     * @param string $path Current path
+     * @param string $highlight String to highlight (in XHTML/CSS)
+     * @param string $type Category type (products/cms)
+     */
+    public static function getPath($url_base, $id_category, $path = '', $highlight = '', $category_type = 'catalog', $home = false)
```

Comment:
Because it seems we deleted a file (not marked as deprecated) without checking if the functions were actually used. @tchauviere moved the concerned function in the Tools file.